quickstart/theme.go

```diff
@@ -0,0 +1,46 @@
+package quickstart
+
+import (
+	"github.com/charmbracelet/huh"
+	"github.com/charmbracelet/lipgloss"
+)
+
+// Temporary file for themes before merging with interactive mode work. This wouldn't live under quickstart
+var theme *huh.Theme
+
+func init() {
```

Comment:
temp just for messing around with styling. These would be abstracted into our own models likely